examples/deadline/All-In-AWS-Infrastructure-Basic/python/package/lib/compute_tier.py

```diff
@@ -117,6 +118,12 @@ def __init__(self, scope: Construct, stack_id: str, *, props: ComputeTierProps,
             user_data_provider=UserDataProvider(self, 'UserDataProvider')
         )
 
+        # This is an example of how you can set up a fleet to be enabled for use with the Session
+        # Manager from the AWS Console. After the fleet deploys, you should be able to go into the
+        # EC2 Instances Console, right click on your worker instance, select "Connect", and then
+        # select the Session Manager and open a shell session in the browser.
+        SessionManagerHelper.grant_permissions_to(self.worker_fleet);
```

Comment:
Might be worth adding this to the RenderQueue & UBL as well.

Reply:
I actually tried it on the render queue and it didn't work. I checked the role's policy and it had the additional actions for SSM, so I figured it was something to do with the way we set these instances up. I can look into it a little more.

Reply:
I think I know why this might not be working. The `grantPrincipal` on the RenderQueue & UsageBasedLicensing constructs are the Roles for the task; whereas the SSM permissions need to be added to the Role for the instance on which the ECS tasks are running.

Reply:
So, that might be a point in favour of having a method on the `RenderQueue` & `UsageBasedLicensing` constructs themselves that adds the session manager permissions to the ECS instance; the method can use this `SessionManagerHelper` under the hood.

Reply:
Alternatively, `SessionManagerHelper.grant_permissions_to(self.render_queue.asg)` ( ref: https://docs.aws.amazon.com/rfdk/api/latest/docs/aws-rfdk.deadline.RenderQueue.html#asg )

But, we'd want to be sure to draw attention to that in the example as the correct way to do it.

Reply:
I swear I tried adding it to the render queue's ASG when I tried before, but I clearly did something wrong as your suggestions have worked for me. I've added configuring the render queue and UBL constructs to the TS and Python basic examples.

I started adding a specific method to do this configuration to the render queue, but it felt cleaner to just show in the example that it should be applied to the ASG property.

Reply:
Perfect. Thanks David!